### PR TITLE
Docs: Update plugin FAQ with versions link

### LINKED ISSUE #23999: Gutenberg plugin repo: Add link to Gutenberg versions in WordPress.
**Is your feature request related to a problem? Please describe.**
Users/developers wonder which Gutenberg plugin is included in a specific WordPress release.

**Describe the solution you'd like**
It would be great to add a link information to https://developer.wordpress.org/block-editor/principles/versions-in-wordpress/ which gives an overview over WordPress major releases and the Gutenberg plugin it contains.

I am thinking someone who has access can add a note to the perhaps FAQ here: https://wordpress.org/plugins/gutenberg/
A note in the FAQ could be. FAQ title: Gutenberg plugin versions included in WordPress. Text: See the following documentation link to get an overview over which Gutenberg plugin version is included into each WordPress major release: (then the versions-in-wordpress link).

Why include it there? It is good to have a note in the Gutenberg plugin at the WP repo in regards to which plugin version is included in which WordPress release. It is a very easy way to access the link to the block editor handbook.

Related to this issue 
https://github.com/WordPress/gutenberg/issues/23344

### PR DESCRIPTION
**Pull request by mkaz**
## Description

Adds a question/answer to the plugin FAQ with a link to the new versions document.

Closes #23999 

## How has this been tested?

Confirm link works.

## Types of changes

Updates the readme.txt file for the plugin including a new question and link to answer.

### CODE REVIEW

**Comment by paaljoachim** [comment]
That looks good, Marcus!

**Review by paaljoachim — APPROVED** [review]

File: readme.txt [hunk 1]
```diff
@@ -50,6 +50,10 @@ The four phases of the project are Editing, Customization, Collaboration, and Mu
 - <a href="https://wordpress.org/gutenberg/handbook/">Documentation: Creating Blocks, Reference, and Guidelines</a>
 - <a href="https://wordpress.org/gutenberg/handbook/reference/faq/">Additional frequently asked questions</a>
 
+= Where can I see Gutenberg plugin versions included in WordPress? =
+
+View <a href="https://developer.wordpress.org/block-editor/principles/versions-in-wordpress/">the Versions in WordPress</a> documentation to get a table for which Gutenberg plugin version is included into each WordPress major release.
```
Comment (annezazu) [inline readme.txt:55]: I might say this instead as it feels (to me) a bit smoother:

"View <a href="https://developer.wordpress.org/block-editor/principles/versions-in-wordpress/">the Versions in WordPress</a> document to get a table showing which Gutenberg plugin version is included in each major WordPress release."

File: readme.txt [hunk 2]
```diff
@@ -50,6 +50,10 @@ The four phases of the project are Editing, Customization, Collaboration, and Mu
 - <a href="https://wordpress.org/gutenberg/handbook/">Documentation: Creating Blocks, Reference, and Guidelines</a>
 - <a href="https://wordpress.org/gutenberg/handbook/reference/faq/">Additional frequently asked questions</a>
 
+= Where can I see Gutenberg plugin versions included in WordPress? =
```
Comment (annezazu) [inline readme.txt:53]: This is nitpicky but I like to be specific: "Where can I see which Gutenberg plugin versions are included in each WordPress release?"

**Review by annezazu — APPROVED** [review]
Just left some minor, non blocking feedback :)

File: readme.txt (hunk 1 above)
Comment (mkaz) [reply readme.txt:55]: :+1:

File: readme.txt (hunk 2 above)
Comment (mkaz) [reply readme.txt:53]: 👍

**Pull request merged**